tests/baselines/reference/docker/office-ui-fabric.log

```diff
@@ -221,32 +139,95 @@ lerna info Executing command in 42 packages: "yarn run build"
+@uifabric/file-type-icons: [XX:XX:XX XM] ▲ One of these [node-sass, postcss, autoprefixer] is not installed, so this task has no effect
+@uifabric/foundation: [XX:XX:XX XM] ▲ One of these [node-sass, postcss, autoprefixer] is not installed, so this task has no effect
+@uifabric/icons: [XX:XX:XX XM] ▲ One of these [node-sass, postcss, autoprefixer] is not installed, so this task has no effect
+office-ui-fabric-react: [XX:XX:XX XM] ▲ One of these [node-sass, postcss, autoprefixer] is not installed, so this task has no effect
+office-ui-fabric-react: [XX:XX:XX XM] x Error detected while running 'ts:esm'
+office-ui-fabric-react: [XX:XX:XX XM] x ------------------------------------
+office-ui-fabric-react: [XX:XX:XX XM] x Error: Command failed: /usr/local/bin/node "/office-ui-fabric-react/node_modules/typescript/lib/tsc.js" --pretty --target es5 --outDir lib --module esnext --project "/office-ui-fabric-react/packages/office-ui-fabric-react/tsconfig.json"
+office-ui-fabric-react:     at ChildProcess.exithandler (child_process.js:295:12)
+office-ui-fabric-react:     at ChildProcess.emit (events.js:223:5)
+office-ui-fabric-react:     at ChildProcess.EventEmitter.emit (domain.js:498:23)
+office-ui-fabric-react:     at maybeClose (internal/child_process.js:1021:16)
+office-ui-fabric-react:     at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
+office-ui-fabric-react: [XX:XX:XX XM] x stdout:
+office-ui-fabric-react: [XX:XX:XX XM] x src/components/Calendar/Calendar.tsx:8:31 - error TS2307: Cannot find module './Calendar.scss'.
+office-ui-fabric-react: 8 import * as stylesImport from './Calendar.scss';
+office-ui-fabric-react:                                 ~~~~~~~~~~~~~~~~~
+office-ui-fabric-react: src/components/Calendar/CalendarDay.tsx:21:31 - error TS2307: Cannot find module './Calendar.scss'.
+office-ui-fabric-react: 21 import * as stylesImport from './Calendar.scss';
+office-ui-fabric-react:                                  ~~~~~~~~~~~~~~~~~
+office-ui-fabric-react: src/components/Calendar/CalendarMonth.tsx:15:31 - error TS2307: Cannot find module './Calendar.scss'.
+office-ui-fabric-react: 15 import * as stylesImport from './Calendar.scss';
+office-ui-fabric-react:                                  ~~~~~~~~~~~~~~~~~
+office-ui-fabric-react: src/components/Calendar/CalendarYear.tsx:5:31 - error TS2307: Cannot find module './Calendar.scss'.
+office-ui-fabric-react: 5 import * as stylesImport from './Calendar.scss';
+office-ui-fabric-react:                                 ~~~~~~~~~~~~~~~~~
+office-ui-fabric-react: src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.base.tsx:70:38 - error TS2554: Expected 1 arguments, but got 2.
+office-ui-fabric-react: 70           {onRenderField(this.props, this._onRenderField)}
+office-ui-fabric-react:                                         ~~~~~~~~~~~~~~~~~~~
+office-ui-fabric-react: src/components/ComboBox/ComboBox.tsx:457:13 - error TS2554: Expected 1 arguments, but got 2.
```

Comment:
Just an fyi @sandersn 

Nestled in among all the scss related errors (which maybe point to a linux build issue; they _do_ do codegen for all their scss files, so it seems like that codegen isn't happening, per the log output about skipping an scss related task above; I'm looking into it) is this error. This error is new form 3.7. I wager this error is correct to issue, however. 
`onRenderField` is essentially of type `((p: Props) => JSX.Element) | ((p?: Props, previousOnRender?: ReferenceToThisWholeType) => JSX.Element)` Previously, we allowed the two parameter call - `onRenderField(this.props, this._onRenderField)` - however now we forbid it, as our signature subtyping rules correctly perform subtype reduction to just `(props: Props) => JSX.Element` (as the minimal required signature) and forbid passing the second argument. It's invalid because I could pass, say, `(props: Props, content?: Context) => JSX.Element` which is assignable to `(props: Props) => JSX.Element`, but would then cause the code to potentially error at runtime (as a render function is passed in instead of the expected `Context`).

Reply:
Errrrr, the sass errors definitely only crop up when I add a new version of TS. Maybe the process of adding the new version is somehow breaking codegen? That's what it looks like.

Reply:
Trying to make a summary of your two comments:

1. sass errors may be because new version of TS fails before the build generates them.
2. the new arity error is correct.

Did I get that right?

Reply:
Something like that, yeah. I'm not wholly sure on 1.; running `yarn add typescript@../typescript.tgz --exact --ignore-scripts` may somehow be breaking the `.sass -> .sass.ts` codegen they have setup, too; I'm not _really_ sure yet. The new arity error seems correct though, yes.